consumer/processing.go

```diff
@@ -161,6 +161,28 @@ func writeRecommendations(
 	return tStored, nil
 }
 
+func (consumer *KafkaConsumer) writeInfoReport(
+	msg *sarama.ConsumerMessage, message incomingMessage, infoStoredAtTime time.Time,
+) error {
+	err := consumer.Storage.WriteReportInfoForCluster(
+		*message.Organization,
+		*message.ClusterName,
+		message.ParsedInfo,
+		infoStoredAtTime,
+	)
+	if err != nil {
+		if err == types.ErrOldReport {
+			logMessageInfo(consumer, msg, message, "Skipping because a more recent info report already exists for this cluster")
+			return nil
+		}
+
+		logMessageError(consumer, msg, message, "Error writing info report to database", err)
+	}
+	logMessageInfo(consumer, msg, message, "Stored info report")
```

Comment:
in case `err != nil`

we will log both 
`"Error writing info report to database"`
And
`"Stored info report"`

you forgot a return?